iothub/device/samples/PnpDeviceSamples/PnpDeviceSamples/SimpleThermostat/Program.cs

```diff
@@ -0,0 +1,90 @@
+﻿using System;
+using System.Text;
+using System.Threading.Tasks;
+using Microsoft.Azure.Devices.Client;
+
+namespace SimpleThermostat
+{
+    public class Program
+    {
+        private const string DeviceConnectionString = "device_connection_string_here";
+        private const string ModelId = "dtmi:com:example:simplethermostat;1";
+
+        private static DeviceClient s_deviceClient;
+
+        public static async Task Main(string[] _)
+        {
+            await RunSampleAsync().ConfigureAwait(false);
+        }
+
+        private static async Task RunSampleAsync()
+        {
+            // This sample follows the following workflow:
+            // -> Initialize device client.
+            // -> Set handler to receive "target temperature" updates.
+            // -> Set handler to receive "reboot" command.
+            // -> Retrieve current "target temperature".
+            // -> Send "current temerature" update over both telemetry and property update.
```

Comment:
temerature = temperature

Reply:
A cool feature is to install an extension on Visual Studio for spell check. I did it last week and it works amazingly well. There will be a pink squiggly line under anything that has a typo - Visual studio spell checker